Class/Warrior/Spells/SunderArmor.cpp

```diff
@@ -0,0 +1,73 @@
+#include "SunderArmor.h"
+
+#include "Buff.h"
+#include "CastComplete.h"
+#include "CharacterStats.h"
+#include "CombatRoll.h"
+#include "CooldownControl.h"
+#include "HeroicStrike.h"
+#include "MainhandAttack.h"
+#include "OffhandAttack.h"
+#include "Warrior.h"
+#include "WarriorSpells.h"
+
+SunderArmor::SunderArmor(Warrior* pchar, WarriorSpells* spells) :
+    // TODO Find the right spell icon
+    Spell("SunderArmor", "Assets/ability/Ability_warrior_decisivestrike.png",
```

Comment:
Checking the protection talent tree it seems icon should be `Assets/ability/Ability_warrior_sunder.png`.

Also add a space in the spell name, as it is currently "SunderArmor" rather than "Sunder Armor".